default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, rustPlatform, fetchFromGitHub, openssl, stdenv, libpulseaudio, pkg-config, protobuf, lame, libopus, git, runCommand, copyPathToStore }:
+
+let
+    src = copyPathToStore ./.;
+    revision = runCommand "get-rev" {
+        nativeBuildInputs = [ git ];
+        dummy = builtins.currentTime;
```

Comment:
Why do we need this?

Reply:
`runCommand` is cached in the store, this makes sure this command is always rerun and the cache version is not used. This prevent some weird behavior I was seeing where this command would not update after I committed something. Will add a comment.